### ISSUE TITLE
Coupled Booking and subsequent visits de-couple inconsistently

### ISSUE THREAD

**Issue by AkubaD** [HRHR-project/e-registry-app#68]
The coupling of the 1st and subsequent visits for ANC, PNC and Newborn care seem to decouple inconsistently. I had observed this once before with @lukasewvd and clearing cache fixed it. But on recent trip to Bangladesh, we had it persist through cache clearing, and re-logging in. There did not seem to be any consistency in what makes this occur: we tried different machines, browsers and users(Monjur's Mac, versus the Chromebook, versus my PC) and got very inconsistent results. Sorry, I can't be more helpful...

![image](https://user-images.githubusercontent.com/36447512/38732598-0ada1ea0-3f1f-11e8-9cb3-d9078b3916dc.png)

**Comment by lukasewvd** [comment]
I have spent some time trying to reproduce this without success. If it happens again write down exactly what you did before the error occurred.

**Comment by AkubaD** [comment]
Hi @lukasewvd 
I have had difficulty reproducing it as well. That is the concern, that it happens inconsistently. What I observed while I was in Bangladesh was that it was generated through a regular login process. As mentioned above, we changed users, relogged in as the same user, changed machines, cleared cache and it would persist inconsistently.

**Comment by Bekkalizer** [comment]
Do we have two issues here? The registration form not always appearing after a cache clean have been observed from time to time, but it has proven very hard to pin down and fix.
The coupling - is it working now?

**Comment by AkubaD** [comment]
Yes there are 2 issues, going on. I put them together because they have been experienced similarly and seem to have the same solution - clear cache and restart. I will put the registration one in a new issue then.
The de-coupling is still an issue, They have been experiencing it consistently in Bangladesh. It has created an issue in training that when they are decoupled and you start an ANC1 visit, and want to create another one, the add button is disabled because ANC1 is not repeatable. The way it is supposed to work is because ANC1 is coupled with ANC subseqent, the add button would be enabled because it is creating an ANCnn visit not an another ANC1. Because the coupling is not working, now you can't add a new ANC visit from ANC1.
 But because this doesn't happen consistently, it is difficult to train these very new users on recognising the decoupling and know what to do if they can't add a follow-up ANC

**Comment by lukasewvd** [comment]
@AkubaD I have made a possible fix (note that I am saying possible as I have not been able to reproduce the bug and therefore also not able to test the fix). There are two things that need to be done by Monjur:

1. Update the app in all instances.
2. Push a config to all instances.

The config can be pushed to the server with this command:
`curl -X POST \
  [**BASEURL**]/api/dataStore/tracker/mainMenuConfig \
  -H 'Authorization: Basic YWRtaW46RGhha2ExMjMh' \
  -H 'Cache-Control: no-cache' \
  -H 'Content-Type: application/json' \
  -d '{
  "bottomLineItems": {
    "BjNpOxjvEj5":true,
    "PUZaKR0Jh2k":true,
    "uD4lKVSbeyB":true,
    "uUHQw5KrZAL":true
  },
  "neverShowItems": {
    "iXDSolqmauJ": true,
    "tlzRiafqzgd": true,
    "w9cPvMH5LaN": true,
    "WPgz41MctSW":true,
    "HaOwL7bIdrs": true,
    "MO39jKgz2VA": true,
    "E8Jetf3Q90U": true
  },
  "topLineStageFilter": {},
  "headerStages": [],
  "headerCombineStages": {
    "w0pwmNYugKX": "dqF3sxJKBls",
    "piRv8jtcLQV": "IlSUGDq9QDc",
    "FRSZV43y35y": "fSE8JyGdsV6",
    "WZbXY0S00lP": "edqlbukwRfQ"
  }
}'`

**NOTE:**
Change BASEURL with the base url of the instance.

**Comment by AkubaD** [comment]
Thanks! I'll follow up with him
